README_AUTOMATED_RELEASE.md

```diff
@@ -0,0 +1,73 @@
+# Automated release
+
+Make releasing easy!
+
+If you have any question, feel free to ask to me (@wakwa3125).
+
+# ATTENTION
+
+**FIRST YOU MUST UPLOAD YOUR APK FILE WITH NON-AUTOMATED FLOW**  
+**THIS IS PUBLISHING API RESTRICT**
+
+# Setup
+
+## 1. Install supply(fastlane)
+
+```
+gem install fastlane
+```
+
+## 2. Create Service Account at GooglePlayConsole.
```

Comment:
Yes right.
But there are many noise to read core sentences.
So I try to rewrite about setup sentences.
But now I think I should add original link of fastlane,'s one.